deepchem/models/torch_models/smiles2vec.py

```diff
@@ -177,3 +182,103 @@ def forward(self, smiles_seqs: List):
         else:
             Logits = Logits.view(-1, self.n_tasks, 1)
             return Logits
+
+
```

Comment:
Please add this model to the .rst docs. Also add it to the model cheatsheet csv file

Reply:
done adding